apps/gateway/test/champions_test.exs

```diff
@@ -54,11 +54,6 @@ defmodule Gateway.Test.Champions do
 
       assert user.username == username
 
-      # CreateUser with the same username fails
-      :ok = SocketTester.create_user(socket_tester, username)
-      fetch_last_message(socket_tester)
-      assert_receive %WebSocketResponse{response_type: {:error, %Error{reason: "username_taken"}}}
```

Comment:
You could create the user and test the opposite instead of just deleting this.

Reply:
Done! i also discovered that the afk_gacha_team created the method `get_user_by_username`, i decided to remove that method since the username is not an id thus is not unique causing that method to explode. I checked both repos of afk_gacha_game and that method is not used so it should be sure to delete

Reply:
Yeah, nice catch. If Gacha game needs that kind of function, it should use `get_user_by_game_id_and_username`

Reply:
yeah! i think if needed it'll be created when the times come